### ISSUE TITLE
pgAdmin in v5

### ISSUE THREAD

**Issue by rahulbreezo** [CrunchyData/postgres-operator#3032]
## Overview

I am trying to add pgAdmin in v5,   though there are no docs for pgAdminin v5 but PR for this seems merged: https://github.com/CrunchyData/postgres-operator/pull/2786

but When I tried to deploy pgAdmin using this :

```
  userInterface:
    pgAdmin:
      dataVolumeClaimSpec:
        accessModes:
          - 'ReadWriteOnce'
        storageClassName: resize-io2
      service:
        type: LoadBalancer
```

I get error: 
`error: error validating "db/patches/staging/": error validating data: ValidationError(PostgresCluster.spec): unknown field "userInterface" in com.crunchydata.postgres-operator.v1beta1.PostgresCluster.spec; if you choose to ignore these errors, turn validation off with --validate=false`, 

So I am not sure its releases in v5 or not,

## Environment

Please provide the following details:

- Platform: (`Kubernetes`, `EKS`)
- PGO Version: `5.0.4`
- Postgres Version `14`

**Comment by ValClarkson** [comment]
Hi @rahulbreezo,
We have recently released PGO v5.1.0, I believe your issue has been resolved with this latest release, please update to PGO v5.1.0 and try again. If your error still persists please feel free to reopen this issue or create a new one.

Please see release notes: [5.1.0](https://access.crunchydata.com/documentation/postgres-operator/5.1.0/releases/5.1.0/)

**Comment by todeb** [comment]
@ValClarkson 
It is a bit weird userInterface is working when you install postgres-operator via 
`kubectl apply --server-side -k kustomize/install/default`
but it is not when postgres-operator is installed via helm:
` error validating data: ValidationError(PostgresCluster.spec): unknown field "userInterface" in com.crunchydata.postgres-operator.v1beta1.PostgresCluster.spec; if you choose to ignore these errors, turn validation off with --validate=false`

CRD postgresclusters.postgres-operator.crunchydata.com installed via helm does not contain "userInterface"
`k get crd postgresclusters.postgres-operator.crunchydata.com -o yaml | grep userInterface`
